### ISSUE TITLE
Keyboard shortcuts for speed of audio

### ISSUE THREAD

**Issue by CircularKen** [edrlab/thorium-reader#1154]
Along with the ability to alter Keyboard shortcuts for "AudioNext", "AudioPlayPause", "AudioPrevious" & "AudioStop" in the GUI and the JSON file it would be great to also have the ability to add and edit Keyboard shortcuts that speed up and slow down audio in both TTS and media overlays (along with an extra shortcut to reset to 1x speed?) and so match the options in the clickable controls.

<img width="331" alt="image" src="https://user-images.githubusercontent.com/17338941/91733585-9e850080-eba1-11ea-9d8f-82fadc20b8a3.png">

**Comment by danielweck** [comment]
Agree. Any suggestions? (please no MacOS-specific “Apple key”, as at the moment we aim to use cross-platform CTRL SHIFT ALT key modifiers, in combination with common keyboard codes)

**Comment by CircularKen** [comment]
As they are configurable then I suppose not so important but I like the idea that they are grouped with other audio controls and can be pressed with one hand. So I suggest CTRL SHIFT Digit 1 for slow down, CTRL SHIFT Digit 2 for 1x speed, CTRL SHIFT Digit 3 for speed up.

**Comment by danielweck** [comment]
Fixed in https://github.com/edrlab/thorium-reader/pull/3622

The delta is now `0.1`, the minimum is `0.2` (inclusive) and the maximum is `6.0` (inclusive). I tested this with TTS readaloud synthetic speech voices on my Mac computer, as well as with pre-recorded audio in EPUB3 Media Overlays publications.

Similarly to the zoom keyboard shortcut (`ctrl` + `0` to reset, `ctrl` + `8` to decrease, `ctrl` + `9` to increase), the audio speed keys are `ctrl` + `7` to reset, `ctrl` + `5` to decrease, `ctrl` + `6` to increase.
